src/framework/core/js/FluidView.js

```diff
@@ -448,6 +448,17 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     /**
+     * Returns the document to which an element belongs, or the element itself if it is already a document
+     *
+     * @param @param {jQuery||Element} element the element to return the document for
+     * @return {Document} dokkument the document in which it is to be found
+     */
+    fluid.getDocument = function (element) {
+        var node = fluid.unwrap(element);
+        return node.nodeType === 9 ? node : node.ownerDocument;
```

Comment:
I'm assuming nodeType of 9 is a document. Could you leave a comment here about that?